kubernetes/templates/mysql/statefulset.yaml

```diff
@@ -64,31 +90,55 @@ spec:
         readinessProbe:
           exec:
             # Check we can execute queries over TCP (skip-networking is off).
```

Comment:
could you explain?

Reply:
This is used to mark the pod as ready.... Kubernetes will try to execute this to check the database is up and mark the pod as ready.... Since i removed the `ALLOW_EMPTY_ROOT_PASSWORD` option, this check requires the mysql which is loaded in an env var.... So to be able to use this env var i need to move the check to a bash command where the env var is available